City-of-Helsinki/helsinki-paatokset
# PP-262: Solve issues with setting cluster settings from env + configs…

Read cluster settings from .env variables without breaking local dev environment
Add block for policymakers search
Configure es container

To test:
- Add `REACT_APP_ELASTIC_URL=https://elastic-helsinki-paatokset.docker.so` to your .env.local file
- Run `make up`. Verify that variable is set by running `make shell` and  then `printenv` inside container
- Just to make sure you get the latest version of the search app, run `rm -rf vendor/paatokset; composer install`
-  Run `drush cr; drush cim -y`. You should not get any errors from elasticsearch settings.
- If you dont have data indexed, go to [https://helsinki-paatokset.docker.so/fi/admin/config/search/search-api/](https://helsinki-paatokset.docker.so/fi/admin/config/search/search-api/) -> go to each index and select 'Index now' from the view tab. Make sure you have data from `paatokset_agenda_items`, `paatokset_meetings` and `paatokset_policymakers` migrations.
- Navigate to [https://helsinki-paatokset.docker.so/fi/paatokset](https://helsinki-paatokset.docker.so/fi/paatokset). You should get results from your searches.
- Navigate to [https://helsinki-paatokset.docker.so/fi/paattajat](https://helsinki-paatokset.docker.so/fi/paattajat). Policymaker search should be at the top of the page and operational.
